README.md

```diff
@@ -77,11 +77,11 @@ The core set of binding data is:
 * **port** - the port to access the service.
 * **endpoints** - the endpoint information to access the service in a service-specific syntax, such as a list of hosts and ports. This is an alternative to separate `host` and `port` properties.
 * **protocol** - the protocol of the service.  Examples: http, https, postgresql, mysql, mongodb, amqp, mqtt, kafka, etc.
-* **contextRoot** - a context root to be used for this service.  Example: the context root for a RESTful service
+* **basePath** - a URL prefix for this service, relative to the host root. It MUST start with a leading slash `/`.  Example: the URL prefix for a RESTful service. 
 * **username** - the username to log into the service.  Can be omitted if no authorization required, or if equivalent information is provided in the password as a token.
 * **password** - the password or token used to log into the service.  Can be omitted if no authorization required, or take another format such as an API key.  It is strongly recommended that the corresponding ConfigMap metadata properly describes this key.
 * **certificate** - the certificate used by the client to connect to the service.  Can be omitted if no certificate is required, or simply point to another Secret that holds the client certificate.  
-* **uri** - for convenience, the full URI of the service in the form of `<protocol>://<host>:<port>[/<contextRoot>]`.
+* **uri** - for convenience, the full URI of the service in the form of `<protocol>://<host>:<port>[basePath>]`.
```

Comment:
should we remove that trailing `>` character?   It looks like a typo from before.

Reply:
I fixed it :)